packages/dropdowns/README.md

```diff
@@ -0,0 +1,107 @@
+# @zendeskgarden/react-dropdowns [![npm version](https://img.shields.io/npm/v/@zendeskgarden/react-dropdowns.svg?style=flat-square)](https://www.npmjs.com/package/@zendeskgarden/react-dropdowns)
+
+This package includes components relating to dropdowns in the
+[Garden Design System](https://zendeskgarden.github.io/).
+
+## Installation
+
+```sh
+npm install @zendeskgarden/react-dropdowns
+
+# Peer Dependencies - Also Required
+npm install react react-dom prop-types styled-components@3.4.10 @zendeskgarden/react-theming
+```
+
+## Requirements
+
+This package uses several modern React features like [Context](https://reactjs.org/docs/context.html)
+and [Hooks](https://reactjs.org/docs/hooks-intro.html) to provide a more consistent developer experience.
```

Comment:
"to provide a more consistent developer experience" doesn't really mean anything to me – consistent compared to what? Maybe "to provide an enhanced component API" instead.

Reply:
I've updated this to your suggestion.